server/layer.go

```diff
@@ -106,7 +106,7 @@ func (l *Layer) Remove() error {
 		return nil
 	}
 
-	ms, err := Manifests()
+	ms, err := Manifests(true)
```

Comment:
Same here?

Reply:
Similar to the above thinking - these call sites are more narrowly scoped around layers that were used, but are transitioning to zero ref counts, and I'm a bit concerned one corrupt manifest will result in excessive garbage accumulation when we are more confident we want to remove the given layer.